docs/rules/default-param-last.md

```diff
@@ -0,0 +1,25 @@
+# enforce default parameters to be last (default-param-last)
+
+Putting default parameter at last allows function calls to omit optional tail arguments.
```

Comment:
I think it might be good to post an example of what happens when optional arguments are not last and someone calls a function thinking the optional parameter is skipped. What do you think?

Reply:
Agreed, an example will be good.